cartographer/mapping/proto/trajectory.proto

```diff
@@ -17,14 +17,17 @@ syntax = "proto2";
 package cartographer.mapping.proto;
 
 import "cartographer/transform/proto/transform.proto";
+import "cartographer/sensor/proto/sensor.proto";
 
 option java_outer_classname = "TrajectoryOuterClass";
 
 message Trajectory {
-  // NEXT_ID: 7
+  // NEXT_ID: 9
   message Node {
     optional int64 timestamp = 1;
     // Transform from tracking to map frame.
+    optional sensor.proto.RangeData range_data = 7;//2D Range data
```

Comment:
I suggest not to add this data here. The problem is that the `Trajectory` proto is used in `SparsePoseGraph` which contains and would thus need to contain all range data in a single proto.

Rather, we should have a separate proto message that contains a node ID and the range data, and we would store many instances of this message in the proto stream. But let's do this in a follow up PR.